indy_node/server/request_handlers/read_req_handlers/get_revoc_reg_handler.py

```diff
@@ -46,8 +46,8 @@ def get_result(self, request: Request):
     def get_current_revoc_entry_and_revoc_def(self, author_did, revoc_reg_def_id, req_id):
         assert revoc_reg_def_id
         current_entry, _, _, _ = self._get_revoc_def_entry(revoc_reg_def_id=revoc_reg_def_id,
```

Comment:
Here is committed needs to False.
A rule:
- when we return a result to the user (get_result in read request handlers) - we do require a committed state
- when we just get the value for some other purposes (validation, checking for existence, etc,) - we require uncommitted state